### ISSUE TITLE
[Bug] uploadFile not working in Linux

### ISSUE THREAD

**Issue by abriginets** [puppeteer/puppeteer#3839]
Hello. I'm expiriencing some issues with `uploadFile()` in Linux. For some reason my code won't work after I call `click()` on `<input type="file">`.Then I moved my project from Ubuntu 18.04 VPS to my home workstation with Windows 10 and tried to run it there and everything worked just fine. I have no idea why this could happen but my code works fine if I comment those lines:

```
var fileInput = await newThread.$('#createbox input[type="file"]')
await fileInput.uploadFile(...filesToUpload)
```

Do you have any ideas on why is this happening? Maybe there are some access rights which might be blocking my files or something? I'm running my script via SSH console with `root` user. Files I need to upload belongs to `root` user and have 0644 oct.

**Tell us about your environment:**

* Puppeteer version: 1.11.0
* Platform / OS version: Ubuntu 18.04
* URLs (if applicable):
* Node.js version: 10.15.0

**Comment by ClassAxion** [comment]
I think that I have the same issue.. 

Uploading files using **fileChooser** or **uploadFile** works fine on my laptop with Manjaro but if I moved my code into Ubuntu 18.04 VPS and run using xvfb then it doesn't work when selecting file.

**Comment by itsmrTech** [comment]
Guys, did you find any solution?

**Comment by paullewis** [comment]
This should be fixed as of 532ae573d29063ffd081ed9ee340d71b25320dec as we changed the underlying implementation to use `DOM.setFileInputFiles`, which behaves a lot more as expected. We also fire events for `change` and `input`, too. Going to close this issue, but please let us know if it's not fixed.

**Comment by GuilhermeZaniniMoreira** [comment]
I think it's still not working on Ubuntu.
